tests/test_preprocess_artifacts.py

```diff
@@ -18,18 +19,36 @@ def __init__(self, tmp_path: pathlib.Path) -> None:
         self.artifact_name: str = "artifact"
         self.artifact_path: pathlib.Path = self.artifact_dir / self.artifact_name
         self.artifact_content: str = "artifact content"
+        self.dir_name: str = "artifacts_test_dir"
+        self.dir_archive: pathlib.Path = self.artifact_dir / f"{self.dir_name}.zip"
 
     def write_config_file(self, artifact_prebuild_clean: bool) -> None:
+        artifact_in_dir = f"{self.dir_name}/{self.artifact_name}"
         config: str = inspect.cleandoc(f"""
             from universum.configuration_support import Configuration, Step
-            step = Step(name='Step',
-                        command=['bash', '-c', 'echo "{self.artifact_content}" > {self.artifact_name}'],
-                        artifacts='{self.artifact_name}',
-                        artifact_prebuild_clean={artifact_prebuild_clean})
-            configs = Configuration([step])
+            step_with_file = Step(name='Step with file',
```

Comment:
I would use the same formatting for step_with_file and step_with_dir claming; but that's a subjective preference.